sambamba_merge

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# arguments are:
+# 1    - number of cores
+# 2    - outfile
+# 3..N - input bams
+
+if [[ "$#" -lt 3 ]]; then
+    echo "not enough parameters specified. Expected numCores, outfile, and input_bams"
+    exit 1
+fi
+#if there is only one bam, just copy it
+if [[ "$#" -eq 3 ]]; then
+    cp $3 $2
```

Comment:
Might need to watch out for quoting here in case something like this happens:
```sh
sambamba_merge 1 'best name ever.bam' some_existing.bam
```
I usually end up making perl wrappers just to avoid dealing with all the shell shenanigans, since Perl is also in the default image.  (Of course, who's putting spaces in their filenames! :smile:)